### ISSUE TITLE
Question about your chart/graph library

### ISSUE THREAD

**Issue by slobanthony** [PhilJay/MPAndroidChart#1831]
Hello,

I'm currently developing an Android application for a client that requires a certain chart mechanism.
While drawing charts within the application is a quite simple task, the problem is that we require the library that can place moveable markers on the charts them selves.
The example of what we need is in the attachment. 
The image shows only one, but we need up to 8 independent markers that can be moved with the finger. And they have to work absolutely smooth on a scale of 4096 values drawn on a chart. The implementation does not have to match the image, any kind of implementation is fine, as long as the result is what we need.
Basically we need multiple moveable markers, that can be moved left and right on the chart and fetch X value upon release.

Does your library support something like this?

Best Regards,
Slobodan
![hxcf3](https://cloud.githubusercontent.com/assets/19684206/15704098/5166cca0-27e9-11e6-996b-7697aacccc7c.jpg)

**Comment by PhilJay** [comment]
Well you can have one marker that is movable. If you want multiple, you will have to modify the library to your needs. It should not be too hard, but it's not something that is supported by default.

**Comment by slobanthony** [comment]
Sorry to bother you still. But any chance of pointing me into the right direction for this movable marker? :)
Your library is quite large, and can't figure out where to look for so I can figure out what needs changing.

Thanks again,
Slobodan

**Comment by PhilJay** [comment]
The code related to drawing the marker can be found here: https://github.com/PhilJay/MPAndroidChart/blob/master/MPChartLib/src/com/github/mikephil/charting/charts/Chart.java#L674-727

What currently happens is that the chart loops over all objects that are set for highlighting and draws the same markerview at that position. This is probably where you need to start.
